supereeg/model.py

```diff
@@ -633,6 +641,7 @@ def _force_update(mo, bo, width=20):
 
     # get _rbf weights
     sub__rbf_weights = _log_rbf(mo.locs, bo.get_locs(), width=width)
+    #sub__rbf_weights = _rbf(mo.locs, bo.get_locs(), width=width)
```

Comment:
remove line 644-- also, is `_rbf` used anywhere now?  if not, we can remove that function

Reply:
removing and rbf already removed